test/ossl_shim/include/openssl/base.h

```diff
@@ -7,8 +7,8 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef OPENSSL_HEADER_BASE_H
-#define OPENSSL_HEADER_BASE_H
+#ifndef OPENSSL_OPENSSL_BASE_H
+#define OPENSSL_OPENSSL_BASE_H
```

Comment:
it is a bit odd that we have OPENSSL_OPENSSL_BASE_H here, is there a reason for?